docs/src/main/paradox/sbt.md

```diff
@@ -20,15 +20,15 @@ akkaGrpcGeneratedSources := Seq(AkkaGrpc.Server)
 What language to generate stubs for is also configurable:
 ```
 // default is Scala only
-akkaGrpcTargetLanguages := Seq(AkkaGrpc.Scala)
```

Comment:
Can we take those snippets from `scripted` tests or from a simple file in `plugin/src/test` that is compiled?

Reply:
Would be nice indeed :)